setuptools/tests/fixtures.py

```diff
@@ -98,7 +98,10 @@ def venv(tmp_path, setuptools_wheel):
     env = environment.VirtualEnv()
     env.root = path.Path(tmp_path / 'venv')
     env.req = str(setuptools_wheel)
-    return env.create()
+    # Don't pass PYTHONPATH from the current environment to the created venv
+    # It only needs its own paths in sys.path
```

Comment:
I think it would be nice to also explain the reasoning here (summarise the PR discussion).

It is not obvious why `PYTHONPATH` cannot be in the environment when doing `env.create()`

Reply:
In this case, when PYTHONPATH is set during `create()` (which actually installs packages, despite the name that does not clearly indicate that), setuptools is not installed (pip says something along the lines of "the same version fo setuptools is already installed, skipping" and does not fail)

Reply:
I believe this also answers the second question from https://github.com/pypa/setuptools/pull/3133#issuecomment-1050081209

Reply:
Here is what happens exactly:

```
(__venv__) [tmp]$ pip list
Package    Version
---------- -------
pip        21.2.3
setuptools 60.9.3

(__venv__) $ pip install .../setuptools-60.9.3-py3-none-any.whl 
Processing .../setuptools-60.9.3-py3-none-any.whl
setuptools is already installed with the same version as the provided wheel. Use --force-reinstall to force an installation of the wheel.

(__venv__) $ echo $?
0

```

Reply:
Amazing! Thank you very much for the clarification @hroncok.
Now everything makes sense 😄